src/rows.jl

```diff
@@ -228,5 +234,24 @@ Base.size(r::Row2) = (length(getnames(r)),)
     s = PointerString(pointer(getbuf(r), getpos(offlen)), getlen(offlen))
     return escapedvalue(offlen) ? unescape(s, gete(r)) : String(s)
 end
-Base.getproperty(r::Row2, nm::Symbol) = getindex(r, findfirst(==(nm), getnames(r)))
 
+Base.propertynames(r::Row2) = getnames(r)
+
+function Base.getproperty(r::Row2, nm::Symbol)
+    @inbounds x = r[getlookup(r)[nm]]
+    return x
+end
+
+@inline Base.@propagate_inbounds function Parsers.parse(::Type{T}, r::Row2, i::Int) where {T}
```

Comment:
These two functions now allow an even more efficient way to parse values when iterating `CSV.Rows`, like:
```julia
for row in CSV.Rows(file)
    int = Parsers.parse(Int64, row, :column1)
end
```
which will avoid the intermediate `String` allocation when doing `row.column1`.